src/LineaStateBridge.sol

```diff
@@ -11,37 +11,35 @@ import { ICrossDomainOwnableLinea } from "./interfaces/ICrossDomainOwnableLinea.
 
 /// @title World ID State Bridge Linea
 /// @author Worldcoin & James Harrison
-/// @notice Distributes new World ID Identity Manager roots to Linea
+/// @notice Distributes new World ID Identity Manager roots to an Linea Stack network
 /// @dev This contract lives on Ethereum mainnet and works for Linea.
 contract LineaStateBridge is Ownable2Step {
     ///////////////////////////////////////////////////////////////////
     ///                           STORAGE                           ///
     ///////////////////////////////////////////////////////////////////
 
-    /// @notice The address of the LinearWorldID contract Linea
+    /// @notice The address of the LinearWorldID contract on any Linea Stack chain
     address public immutable lineaWorldIDAddress;
 
-    /// @notice Address of the Linea Message Service contract on L1 (Ethereum)
+    /// @notice address for Linea Stack chain Ethereum mainnet L1 Message Service contract
     address internal immutable messageServiceAddress;
 
     /// @notice Ethereum mainnet worldID Address
     address public immutable worldIDAddress;
 
-    /// @notice Amount of fee on Linea for propagateRoot
-    uint256 internal _feePropagateRoot;
+    /// @notice Amount of gas purchased on the Linea Stack chain for propagateRoot
+    uint32 internal _gasLimitPropagateRoot;
 
-    /// @notice Amount of fee on Linea for SetRootHistoryExpiry
-    uint256 internal _feeSetRootHistoryExpiry;
+    /// @notice Amount of gas purchased on the OLinea Stack chain for SetRootHistoryExpiry
+    uint32 internal _gasLimitSetRootHistoryExpiry;
 
-    /// @notice Amount of fee on Linea for transferOwnershipLinea
-    uint256 internal _feeTransferOwnership;
+    /// @notice Amount of gas purchased on the Linea Stack chain for transferOwnershipLinea
+    uint32 internal _gasLimitTransferOwnership;
 
-    /// @notice Amount of fee on Linea for setMessageService
-    uint256 internal _feeSetMessageService;
+    /// @notice The default gas limit amount to buy on an Linea stack chain to do simple transactions
 
-    /// @notice The default fee for Linea transactions.
-    /// Setting this to 0 means that messages will have to be claimed manually on L2
-    uint256 public constant DEFAULT_LINEA_FEE = 0;
+    /// @dev For estimation see https://github.com/kfastov/worldcoin-bridge-linea/pull/9
+    uint32 public constant DEFAULT_LINEA_GAS_LIMIT = 1_000_000;
```

Comment:
Sorry but this source file was changed in the mainstream and it seems your PR rolls back that changes. Please check the main branch and try to merge it carefully

Reply:
I think I have resolved this